packages/svelte/types/index.d.ts

```diff
@@ -1108,18 +1108,18 @@ declare module 'svelte/compiler' {
 	 * https://svelte.dev/docs/svelte-compiler#svelte-preprocess
 	 * */
 	export function preprocess(source: string, preprocessor: PreprocessorGroup | PreprocessorGroup[], options?: {
-		filename?: string | undefined;
+		filename?: string;
 	} | undefined): Promise<Processed>;
 	export class CompileError extends Error {
 		
 		constructor(code: string, message: string, position: [number, number] | undefined);
 		filename: string | undefined;
 		
-		position: CompileError_1['position'];
+		position: CompileError_1["position"];
```

Comment:
why switch to double quotes everywhere? or is this generated?

Reply:
generated